src/java.base/share/classes/jdk/internal/foreign/AbstractMemorySegmentImpl.java

```diff
@@ -74,7 +74,9 @@
     final MemorySession session;
 
     @ForceInline
-    AbstractMemorySegmentImpl(long length, boolean readOnly, MemorySession session) {
+    AbstractMemorySegmentImpl(long length,
```

Comment:
Here I'm a bit puzzled. Looking at other JDK code, I don't see this rule being applied consistently. I think I'm ok with breaking up long parameter lists (an example of that below) - but doing that for all method decls seems a bit too much IMHO.

Reply:
Agree, we should follow the style used in the rest of the jdk code, at least in java.base.

Reply:
The principle used in this PR is the same as in `java.util.stream.Stream` where methods with 0 <= x <= 2 parameters are on the same line and methods with > 2 parameters are on separate vertical lines.

Reply:
But that does not make it the norm. The majority of code in java.base does not do this. Even in `java.util.stream.Stream` it is not done for every method (e.g. `iterate` puts 3 parameters on the same line).

Reply:
It is true that this does not make it the norm. It is just an explanation of the principle used in this PR.